### PR TITLE
Logging Acceptance Tests

### PR DESCRIPTION
**Pull request by blowmage**
Here are the acceptance tests for Logging. There is a permissions issue modifying sinks. I was able to modify sinks using my user account, but not a service account.

### CODE REVIEW

**Comment by blowmage** [comment]
Looks like the permissions issue is a known issue, as gcloud-node states in their documentation. Only Sinks are affected. Metrics and Entries can be managed with a service account.

I'll leave the Sink tests skipped for now, until we can run them with a service account.

File: acceptance/logging/logging_test.rb [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+      logging.metrics(max: 1).length.must_equal 1
+
+      metric.delete
+
+      logging.metric("#{prefix}-metric").must_be :nil?
+    end
+  end
+
+  describe "Entries" do
+    let(:proto_payload) { Google::Protobuf::Any.new type_url: "example.com/Greeter/SayHello",
```
Comment (quartzmo) [inline acceptance/logging/logging_test.rb]: Would it be better to use a more specific and realistic Protobuf type here? (Note: I'm not sure what that would be, just asking!)

Comment (blowmage) [inline acceptance/logging/logging_test.rb]: I don't think we need a more realistic protobuf object, any `Google::Protobuf::Any` object will do. If you look at what is returned from `Project#entries`, you'll see plenty of log entries that have protobuf payloads. AFAICT the Logging service just passes those values around, it doesn't care what they are.

**Comment by quartzmo** [comment]
Would it be fairly easy to retrieve and verify the entries created with the Ruby Logger impl?

File: acceptance/logging/logging_test.rb (hunk 1 above)
Comment (quartzmo) [inline acceptance/logging/logging_test.rb]: Ok

**Comment by quartzmo** [comment]
Great to see how doing these tests led you to implement GRPCUtils.

**Comment by blowmage** [comment]
Updated to verify the log entries are written. The check has a retry with an incremental backoff, so it should be more stable on the eventual nature of the cloud services.

**Comment by quartzmo** [comment]
Great! Merging!

**Pull request merged**